harness/features/initialize.local.test.ts

```diff
@@ -0,0 +1,153 @@
+import {
+    getConnectionStringForProxy,
+    forEachSDK,
+    createClient,
+    callOnClientInitialized,
+    describeIf,
+    wait
+} from '../helpers'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import { Capabilities, SDKCapabilities } from '../types'
+import nock from 'nock'
+
+jest.setTimeout(10000)
+
+describe('Initialize Tests - Local', () => {
+    const mockServerUrl 
+        = `http://${process.env.DOCKER_HOST_IP ?? 'host.docker.internal'}:${global.__MOCK_SERVER_PORT__}`
+    const scope = getServerScope()
+
+    forEachSDK((name: string) => {
+        const capabilities: string[] = SDKCapabilities[name]
+        const sdkKey = 'server_SDK_KEY'
+        let url: string
+
+        beforeAll(async () => {
+            url = getConnectionStringForProxy(name)
+        })
+
+        afterEach(() => {
+            nock.cleanAll()
+        })
+
+        describeIf(capabilities.includes(Capabilities.local))(name, () => {
+            it('should error when SDK key is missing', async () => {
+                const clientId = uuidv4()
+                scope
+                    .get(`/client/${clientId}/config/v1/server/${sdkKey}.json`)
+                    .reply(200, {})
+
+                const response = await createClient(url, clientId)
+                const { exception } = await response.json()
+                await wait(500)
```

Comment:
I ended up not needing to wait, you might be able to just remove the waits

Reply:
For the first 2 tests the wait needs to be there so that the sdk has a chance fetch the config in case the error checks fail. Otherwise the test would report a false positive. The rest of the tests fail if I don't wait.

I think the reason you're tests pass is because in [handleLocation](https://github.com/DevCycleHQ/test-harness/blob/main/proxies/nodejs/handlers/location.ts#L57) `invokeCommand` would return a `Promise<DVCVariableInterface>` and we `await` it, so the proxy does not respond until after it fetches the variable from the mock endpoint.

I think the logic in `handleLocation` is incorrect because right now it awaits all skd methods regardless of weather you send the `isAsync` param

Reply:
for this test do you need to wait for a config to come back if the sdk key is supposed to error it immediately?

Reply:
hmm I guess not. If the skd fails to throw then the error message won't be there so that's a for sure way to check. I'll remove the wait and also the mock config endpoint